Tests/Functional/Services/CurlTest.php

```diff
@@ -102,4 +102,54 @@ public function setContentType() {
         $this->assertInstanceOf('Symfony\Component\HttpFoundation\Response', $response);
         $this->assertSame('application/json', $response->headers->get('Content-Type'));
     }
+    
```

Comment:
I think the http_parse_headers function from the pecl extension should not be tested. We can create a second function with the own http_parse_headers function which will be tested. In the main function there is only the if function exists.